tensorbay/opendataset/RP2K/loader.py

```diff
@@ -51,12 +52,23 @@ def RP2K(path: str) -> Dataset:
         catagories = os.listdir(segment_path)
         catagories.sort()
         for catagory in catagories:
-            if not os.path.isdir(catagory):
+            catagory_path = os.path.join(segment_path, catagory)
+            if not os.path.isdir(catagory_path):
                 continue
-            image_paths = glob(os.path.join(segment_path, catagory, "*.jpg"))
+            image_paths = _glob(catagory_path, ("*.jpg", "*.png"))
             for image_path in image_paths:
                 data = Data(image_path)
                 data.label.classification = Classification(catagory)
                 segment.append(data)
 
     return dataset
+
+
+def _glob(directory: str, patterns: Tuple[str, str]) -> List[str]:
```

Comment:
generally input type should be more abstract, output type should be more specific
So the input type should be `Iterable[str]`